notebooks/tel_and_site/subsys_req_ver/m1m3/LVV-T232-analysis_actuators.ipynb

```diff
@@ -0,0 +1,397 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    def actuatorLayout(ax, FATABLE):
</pre><p>Can you add docstring to the functions? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/45/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Done